+lumofile/write_NIRS.m

```diff
@@ -276,12 +276,10 @@
   end
   
 else
-  
-  
+    
   % No events recorded
-  s = [];
-  CondNames = {};
-  
+  s = zeros([size(t, 1) 1]); % Add a zeros stimulus matrix for compatibility with Homer2
```

Comment:
Please align short comments with whitepsace, or better, add concise details in a block comment following from 'No events recorded'. Don't describe what's obvious from the code.